Makefile

```diff
@@ -417,7 +417,7 @@ docker-build-ghost-debug:
 
 PHONY: docker-build-bootstrap
 docker-build-bootstrap:
```

Comment:
can we also add to `docker-build-bootstrap-transit`

Reply:
`docker-build-bootstrap-transit` already has a `--build-arg VERSION=$(VERSION)` arg